src/validation.cpp

```diff
@@ -658,6 +639,19 @@ bool MemPoolAccept::PreChecks(ATMPArgs& args, Workspace& ws)
             // Otherwise assume this might be an orphan tx for which we just haven't seen parents yet
             return state.Invalid(TxValidationResult::TX_MISSING_INPUTS, "bad-txns-inputs-missingorspent");
         }
+        // Sanity check each input.
```

Comment:
nit: Could explain a bit more what this sanity check is doing? Maybe a oneline summary of https://github.com/bitcoin/bitcoin/pull/20834#issuecomment-754652998 ?

Reply:
Made big effort to improve the documentation :D